build.gradle

```diff
@@ -24,6 +25,13 @@ configure(subprojects) {
         mavenCentral()
     }
 
+    tasks.withType(FindBugs) {
+        ignoreFailures true
+        reports {
+            xml.withMessages true
```

Comment:
Can we generate report in more user friendly format? It seems only output xml data for now.

Reply:
Done